src/objects.rs

```diff
@@ -345,6 +345,16 @@ impl Id<StopArea> for StopPoint {
 }
 impl_codes!(StopPoint);
 
+pub type Date = String;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum ExceptionType {
+    #[serde(rename = "1")] Add,
+    #[serde(rename = "2")] Remove,
+}
+
+pub type CalendarDatesT = Vec<(Date, ExceptionType)>;
```

Comment:
You can remove the final T

Reply:
T is not for Type?

Reply:
That was because it was clashing with the trait name.